src/main/java/teammates/ui/controller/InstructorCommentsPageData.java

```diff
@@ -126,4 +195,95 @@ public boolean isInstructorAllowedForPrivilegeOnComment(CommentAttributes commen
             return false;
         }
     }
+
+    public void init(Boolean isViewingDraft, Boolean isDisplayArchive, String courseId,
+                                    String courseName, List<String> coursePaginationList,
+                                    Map<String, List<CommentAttributes>> comments,
+                                    String string, InstructorAttributes instructor, CourseRoster roster,
```

Comment:
Its possible for the instructor to be null, if is he "is viewing draft", so I wouldn't be able to use instructor.email. Although I'm not sure how viewing draft functionality works as it hasn't been implemented yet